src/eda_server/api/activation.py

```diff
@@ -216,6 +216,21 @@ async def delete_activation(
     await db.commit()
 
 
+@router.get(
+    "/api/activations/{activation_id}/job_instances",
+    response_model=List[schema.ActivationJobInstance],
+    operation_id="list_activation_job_instances",
+)
+async def list_activation_job_instances(
```

Comment:
What's the point of returning only list of job instance IDs?

Reply:
I tried to follow the format that we already have for [Activation Instance Job Instances.](https://github.com/ansible/eda-server/blob/cbd41655e3ffe7da886a256ad8bfc2b1e3ad6504/src/eda_server/api/activation.py#L415) Is it wrong?